src/renderers/dom/__tests__/ReactDOMProduction-test.js

```diff
@@ -178,6 +178,9 @@ describe('ReactDOMProduction', () => {
     expect(function() {
       class Component extends React.Component {
         render() {
+          // FIXME: undefined is a valid return type in Fiber, at least in the
+          // current implementation. It's treated as empty. I don't know why we
+          // sometimes get false positives.
```

Comment:
From a skim of the jest source it looks like it should be okay, and if I add a log in the ReactDOMFeatureFlags mock factory in test-framework-setup, then it does log many times when I run a test that resets the registry using REACT_DOM_JEST_USE_FIBER=1.